I1463 open shift connector - ready for the review (#1080)
codenvy/che-docs: src/main/pages/che-7/extensions/assembly_openshift-connector-overview.adoc

---
title: OpenShift Connector
keywords:
tags: []
sidebar: che_7_docs

> permalink: che-7/openshift-connector-overview/

Reviewer: The permalink should mirror the title.

folder: che-7/extensions
summary:
---

// Upstream only

ifdef::context[:parent-context-of-openshift-connector-overview: {context}]

[id="openshift-connector-overview_{context}"]
= OpenShift Connector overview

:context: openshift-connector-overview

> OpenShift Connector, also referred to as Visual Studio Code OpenShift Connector for Red Hat OpenShift, is a plug-in for {prod-short} that provides a convenient and intuitive method for interacting with Red Hat OpenShift 3 or 4 clusters.

Reviewer: scratch "convenient" and "intuitive" -- that's marketing talk :)

> OpenShift Connector makes it possible to create, build, and debug applications in the {prod-short} IDE and then deploy the applications directly to the running OpenShift cluster.

Reviewer: ```suggestion
OpenShift Connector makes it possible to create, build, and debug applications in the {prod-short} IDE and then deploy the applications directly to a running OpenShift cluster.
```

> OpenShift Connector acts as a GUI for the OpenShift Do (ODO) utility, which aggregates OpenShift CLI (OC) commands into compact units. As such, OpenShift Connector helps new developers who do not have OpenShift background with creating applications and running them on the cloud. Instead of using several `oc` commands, the user creates a new component or service by selecting a preconfigured template, such as a Project, an Application, or a Service; then and deploys it as an OpenShift Component to their cluste

Reviewer: ```suggestion
OpenShift Connector is a GUI for the OpenShift Do (`odo`) utility, which aggregates OpenShift CLI (`oc`) commands into compact units. As such, OpenShift Connector helps new developers who do not have OpenShift background with creating applications and running them on the cloud. Instead of using several `oc` commands, the user creates a new component or service by selecting a preconfigured template, such as a Project, an Application, or a Service, and then deploys it as an OpenShift Component to their cluster.
```

> This section provides information about installing, enabling, and basic use of OpenShift Connector plug-in.

Reviewer: ```suggestion
This section provides information about installing, enabling, and basic use of the OpenShift Connector plug-in.
```

* link:{site-baseurl}che-7/features-of-openshift-connector[Features of OpenShift Connector]

> * link:{site-baseurl}che-7/installing-openshift-connector-in-eclipse-che[Installing OpenShift Connector in Eclipse Che]

Reviewer: ```suggestion
* link:{site-baseurl}che-7/installing-openshift-connector-in-eclipse-che[Installing OpenShift Connector in {prod}]
```

> * link:{site-baseurl}che-7/authenticating-with-openshift-connector-in-eclipse-che[Authenticating with OpenShift Connector in Eclipse Che]

Reviewer: ```suggestion
* link:{site-baseurl}che-7/authenticating-with-openshift-connector-in-eclipse-che[Authenticating with OpenShift Connector in {prod}]
```

> * link:{site-baseurl}che-7/creating-components-with-openshift-connector-in-eclipse-che[Creating Components with OpenShift Connector in Eclipse Che]

Reviewer: ```suggestion
* link:{site-baseurl}che-7/creating-components-with-openshift-connector-in-eclipse-che[Creating Components with OpenShift Connector in {prod}]
```

> * link:{site-baseurl}che-7/connecting-source-code-from-guthub-to-a-openshift-component-using-openshift-connector[Connecting source code from GitHub to a OpenShift Component using OpenShift Connector]

Reviewer: ```suggestion
* link:{site-baseurl}che-7/connecting-source-code-from-guthub-to-a-openshift-component-using-openshift-connector[Connecting source code from GitHub to an OpenShift Component using OpenShift Connector]
```
